### ISSUE TITLE
Add "ReLU6, HardSwish, KAF, ELiSH, HardELiSH"

### ISSUE THREAD

**Issue by marload** [tensorflow/addons#2045]
**Describe the feature and the current behavior/state.**

These activation functions are required by communities and are commonly used by up-to-date architectures.

- ReLU6
- HardSwish
- KAF
- ELiSH
- HardELiSH

**Relevant information**
- Are you willing to contribute it (yes/no): YES
- Are you willing to maintain it going forward? (yes/no): YES
- Is there a relevant academic paper? (if so, where): YES

[ReLU6](https://arxiv.org/pdf/1704.04861v1.pdf)
[HardSwish](https://arxiv.org/abs/1905.02244v5)
[KAF](https://arxiv.org/pdf/1707.04035v2.pdf)
[ELiSH, HardELiSH](https://arxiv.org/pdf/1808.00783v1.pdf)

- Is there already an implementation in another framework? (if so, where): `torch`, `tf.nn`
- Was it part of tf.contrib? (if so, where): NO

**Which API type would this fall under (layer, metric, optimizer, etc.)**

`tfa.activation`

**Who will benefit with this feature?**

Researchers need up-to-date Activation

**Any other info.**

**Comment by AakashKumarNain** [comment]
`relu6` and `swish` are already  in TF core. I haven't heard any of the others. Can you point out any papers using them?

**Comment by marload** [comment]
As far as I know, HardSwish is not in Core. I think the link I attached will help you explain other Activation Function.

Thank You!

**Comment by bhack** [comment]
HardSwish Is in Tensorflow lite

**Comment by AakashKumarNain** [comment]
Thanks for the info. IMO, it is okay to add new activation functions but we also need to check the coverage, like how many times the paper for a corresponding activation has been cited. If the coverage for a any functionality is very low, it doesn't add much value to the ecosystem but rather increase the burden of maintenance. 

Anyways, that's just my opinion. Let's hear what other maintainers have to say about this. cc: @seanpmorgan @WindQAQ

**Comment by bhack** [comment]
@AakashKumarNain we was thinking something similar in https://github.com/tensorflow/community/pull/241/files#diff-24cf1a9d17b0988437751c2472f2c296R13 but it is still not expanded. 
Probably we could expand on that and just lower the barrier for TFA.

**Comment by AakashKumarNain** [comment]
@bhack I agree that the barrier for TFA should be low as compared to core TF. But we need to decide that factor IMO to make it more clear for the contributors

**Comment by bhack** [comment]
Yes what I meant is that if we could clear up this in the upstreaming process it could be easier to define for TFA (or the other way around).

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
